cli/jobs/single-step/r/iris/src/train.R

```diff
@@ -8,21 +17,25 @@ parser <- add_option(parser, "--data_folder",
 
 args <- parse_args(parser)
 
-cat("data folder...\n")
+print("data folder...\n")
 print(args$data_folder)
 
 file_name = file.path(args$data_folder)
```

Comment:
rongduan had opened a separate PR to correct the file path here, see: https://github.com/Azure/azureml-examples/pull/868/files

can you validate that fix and just update it as part of your pr here?

Reply:
I validated with what is in the example and it works without any change needed. The input that this examples is the https url for iris data and so no need to append file name.